python/ray/autoscaler/_private/local/config.py

```diff
@@ -69,12 +71,19 @@ def prepare_manual(config: Dict[str, Any]) -> Dict[str, Any]:
     return config
 
 
+def get_temp_path() -> str:
+    ray_temp = os.path.join(tempfile.gettempdir(), "ray")
+    if not os.path.exists(ray_temp):
+        os.makedirs(ray_temp)
+    return ray_temp
```

Comment:
Hey @carlogrisetti This looks like a good thing to change! I think the functionality of `get_temp_path` is already present in https://github.com/ray-project/ray/blob/1efcf5c3d5e304db625a085cd40ff003ab99dc63/python/ray/_private/utils.py#L48

Could you refactor to directly use that function?